src/gc/gcimpl.h

```diff
@@ -274,6 +274,13 @@ class GCHeap : public IGCHeapInternal
 
     virtual void DescrGenerationsToProfiler (gen_walk_fn fn, void *context);
 
+    virtual void SetDirty(void* address, size_t size);
+
+    virtual void* GetTable();
```

Comment:
@jkotas If I'm understanding the code correctly, it seems to me that `SetDirty` and `GetTable` and other write-watch related things are unique in that they are the EE calling into the GC, while `StompWriteBarrier*` is the GC calling into the EE - do you think we should have a single method on both `IGCHeap` and `GCToEEInterface` that is responsible for these write-barrier interactions?